src/main/scala/edu/berkeley/ce/rockslicing/Block.scala

```diff
@@ -63,6 +63,23 @@ object Block {
     }
   }
 
+  /** 
+    * Determines whether the input point is within the input block
+    * @param point The point as a tuple
+    * @param block The block that the point is being checked agains
+    * @return True if the point is within the block, false otherwise
+    */
+  private def pointInBlock(point: (Double, Double, Double), faces: Seq[Face]): Boolean = {
```

Comment:
Checkout what I changed to make sure it's what you meant. Flipped to function such that it checks for points outside the block.